### PR TITLE
Add custom LaTeX delimiters to preferences

### PR DESCRIPTION
**Pull request by Paalon**
See [https://github.com/BoostIO/Boostnote/issues/1201](https://github.com/BoostIO/Boostnote/issues/1201). This add optional LaTeX delimiters. 

However, currently this has a bug that LaTeX preference value seems not to be saved but I cannot understand. I want help to solve.

### CODE REVIEW

File: browser/lib/markdown.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ import markdownit from 'markdown-it'
 import emoji from 'markdown-it-emoji'
 import math from '@rokt33r/markdown-it-math'
 import _ from 'lodash'
+import ConfigManager from 'browser/main/lib/ConfigManager'
 
 // FIXME We should not depend on global variable.
 const katex = window.katex
+var config = ConfigManager.get()
```
Comment (sosukesuzuki) [inline browser/lib/markdown.js]: Please use `const` instead of `var`.
In a modern browser, please use `let` or `const`.

**Comment by Paalon** [comment]
The bug was fixed. That was only my mistake of property names.

**Comment by sosukesuzuki** [comment]
Thanks for your fixing! 👍

**Comment by Paalon** [comment]
The change of delimiters seems not reflected unless restarting.

**Pull request merged**